workflow/run_analysis.rb

```diff
@@ -77,29 +77,14 @@ def run_workflow(yml, in_threads, measures_only, debug_arg, overwrite, building_
 
   Dir.mkdir(results_dir)
 
-  # Create lib folder
-  lib_dir = File.join(thisdir, '../lib')
-  resources_dir = File.join(thisdir, '../resources')
-  housing_characteristics_dir = File.join(buildstock_directory, project_directory, 'housing_characteristics')
```

Comment:
Doesn't appear the `buildstock_directory` variable gets used anymore.

Reply:
Yeah, it's not required anymore for run_analysis.